core/sail/extensible-store/pom.xml

```diff
@@ -46,7 +46,9 @@
                     <groupId>org.asynchttpclient</groupId>
                 </exclusion>
             </exclusions>
-        </dependency>
+			<scope>provided</scope>
```

Comment:
As a temporary fix this is fine, but perhaps it should be optional instead (see https://stackoverflow.com/questions/40393098/when-to-use-optionaltrue-optional-and-when-to-use-scopeprovided-scope) - the difference is mostly semantic, but optional is more intended to say "it's up to you as a user to decide if you need/want this thing", whereas provided scope is more for things that are just _expected_ to be present in the runtime environment (e.g. for a webapp, you could have the servlet-api jar as provided). 

Also: I haven't looked at the code, but can the extensible store still operate if this library is _not_ available? Or will it just crash with a ClassNotFoundException? Don't bother if this is a massive effort but if we can do a simple check and have a fallback that (no matter how ineffecient) at least _works_, it makes the "optional" story a bit more convincing :)

Reply:
Changed it to optional.